src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -14,9 +14,9 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import base64
+import gpgkeyverifier
```

Comment:
Importing gpgkeyverifier doesn't do anything here, and there are a number of other places you do this. Is there a specific reason you want to import modules the code doesn't reference?